### DISCUSSION TITLE
[A-L] (2023/24) Foglio 11 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#333] [Esercizi - Viaggi/Piazza] [answered]
![Screenshot 2023-11-23 103955](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/d307a6df-044a-4f35-b081-c56f5f7de0d6)

**Comment by Elia-Belli** [comment]
Dobbiamo verificare che:
- $(\underline v_1, \underline v_2,\underline v_3)$ sono linearmente indipendenti: ciò avviene se $x\underline v_1+y\underline v_2+z\underline v_3=0\iff x=y=z=0$, allora risolviamo il seguente sistema e verifichiamo che la soluzione nulla è l'unica

$$\begin{cases} x+3y-z=0\\
2x+y-z=0\\
2x-y=0
\end{cases}
\iff \begin{pmatrix} 1 & 3 & -1\\
2 & 1 & -1\\
2 & -1 & 0\end{pmatrix}\begin{pmatrix}0 \\
 0\\
 0\end{pmatrix}\to \begin{pmatrix} 1 & 3 & -1\\
0 & -5 & 1\\
0 & 0 & 17/5\end{pmatrix}\begin{pmatrix}0 \\
 0\\
 0\end{pmatrix}\\iff\begin{cases} x=0\\
y=0\\
z=0
\end{cases}$$

- Inoltre sappiamo che una base di $\mathbb{R}^3$ ha cardinalità $3\Rightarrow (\underline v_1, \underline v_2,\underline v_3)$ sono una base

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione Ufficiale
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/902f2d01-dd01-4ad2-b72f-0fe09fc12167)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/e9b3f858-c35e-480e-a24e-0db54733be65)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/27f3e991-389d-4783-9328-15cdc308b047)
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4031ca5b-d7a7-47eb-825e-63c7b69f2e6d)

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/424af7af-e605-48aa-9789-ab4fedbcaa7a)
